src/resources/postcss/dialog.pcss

```diff
@@ -184,4 +184,38 @@ Adding these here for now until common-styles are live
 		justify-content: flex-end;
 	}
 
+	/* -------------------------------------------------------------------------
+	 * Theme Overrides - Avada
+	 * ------------------------------------------------------------------------- */
+
+	 .tribe-theme-avada & {
+
+		div.tribe-dialog {
+			z-index: 99999;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Theme Overrides - Divi
+	 * ------------------------------------------------------------------------- */
+
+	 .tribe-theme-divi & {
+
+		div.tribe-dialog {
+			z-index: 99999;
+		}
+	}
+
+	/* -------------------------------------------------------------------------
+	 * Theme Overrides - Enfold
+	 * ------------------------------------------------------------------------- */
+
+	#top.tribe-theme-enfold &,
+	#top .main_color & {
+
+		div.tribe-dialog {
+			z-index: 99999;
```

Comment:
Can we get a variable for that? Possibly something related to theme compatibility.